app/models/classification.rb

```diff
@@ -149,6 +147,24 @@ def to_s
     name
   end
 
+  def policy_content_ids
+    classification_policies.map(&:policy_content_id)
+  end
+
+  def policy_content_ids=(content_ids)
+    self.classification_policies = content_ids.map do |content_id|
+      ClassificationPolicy.new(policy_content_id: content_id)
+    end
+  end
+
+  def policies
+    Future::Policy.from_content_ids(policy_content_ids)
+  end
+
+  def can_be_related_to_policies?
```

Comment:
Those `can_DO_SOMETHING?` predicate checks are used in the edition classes because of the fact that different STI-types mix in different behaviours. I'm not clear on how making the `Topic` class consistent with that interface makes any sense. This method never gets called and the only scenarious I can imagine where having this method might be useful is if editions and topics end up being edited with the same forms. I really don't think it should be here.